README.rst

```diff
@@ -50,7 +50,8 @@ DNS lookup...
   the lookup.
 * If ``gethostbyname`` does not have it cached then a request is made to the
   known DNS server that was given to the network stack. This is typically the
-  local router or the ISP's caching DNS server
+  local router or the ISP's caching DNS server.
+* Port 53 is opened to send UDP request to DNS server.
```

Comment:
Should this maybe mention that if the request might become a TCP request if it's too large / other details?